python/pyarrow/tests/test_serialization.py

```diff
@@ -662,6 +666,62 @@ def test_scipy_sparse_csr_matrix_serialization():
     assert np.array_equal(sparse_array.toarray(), result.toarray())
 
 
+@pytest.mark.parametrize('tensor_type', tensor_types)
+@pytest.mark.parametrize('index_type', index_types)
+def test_sparse_csc_matrix_serialization(index_type, tensor_type):
+    tensor_dtype = np.dtype(tensor_type)
+    index_dtype = np.dtype(index_type)
+    data = np.array([8, 2, 5, 3, 4, 6]).astype(tensor_dtype)
+    indptr = np.array([0, 2, 3, 4, 6]).astype(index_dtype)
+    indices = np.array([0, 2, 5, 0, 4, 5]).astype(index_dtype)
+    shape = (6, 4)
+    dim_names = ('x', 'y')
+
+    sparse_tensor = pa.SparseCSCMatrix.from_numpy(data, indptr, indices,
+                                                  shape, dim_names)
+
+    context = pa.default_serialization_context()
+    serialized = pa.serialize(sparse_tensor, context=context).to_buffer()
+    result = pa.deserialize(serialized)
```

Comment:
Don't you check the result type?

Reply:
Added.

Reply:
Thanks. I also meant the type in the Python sense, e.g. `assert isinstance(pa.SparseCSCMatrix, result)`.
(who knows, perhaps there's a bug and you're getting back a CSR matrix)

Reply:
Added.